modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc

```diff
@@ -3,7 +3,7 @@
 
 The home page of {webapp} lists all the {walkthrough-name}s from all the Git repositories you are subscribed to. Any user can add {walkthrough-name}s to your cluster.
```

Comment:
Slightly silly suggestion, but I think there might be too many instances of 'all'

```suggestion
The home page of {webapp} lists the {walkthrough-name}s from all the Git repositories you are subscribed to. Any user can add {walkthrough-name}s to your cluster.
```